db/migrate/20171031104602_make_external_id_unique.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+class MakeExternalIdUnique < ActiveRecord::Migration[5.1]
+  class User < ActiveRecord::Base
```

Comment:
Whats this about again?

Is this to skip all the validation? Or are we not loading the rails models for some reason?

Reply:
loading rails models in migrations is dangerous since migrations will run at a random future version of the codebase ... so the code might have changed to use different sorting/scopes or depend on certain columns to already be there